### ISSUE TITLE
Pressing any key exits the script.

### ISSUE THREAD

**Issue by bpekalp** [phil294/AHK_X11#42]
ahk_x11> [xcb] Unknown request in queue while dequeuing
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
AppRun.wrapped: ../../src/xcb_io.c:175: dequeue_pending_request: Assertion `!xcb_xlib_unknown_req_in_deq' failed.
[1]    27390 IOT instruction (core dumped)
```ahk
*XButton1::
    Loop
    {
        GetKeyState, state, XButton1, P
        If state = U
            Break
        Send, {Space}
        Sleep, 20
    }
```

**Comment by bpekalp** [comment]
> Isn't that the same script from before where you said that it worked as expected?

Yes.. Script works but I noticed it aborts when I press anything on my keyboard. Insert, Numlock, Backspace, letters, numbers, function keys etc. It doesn't matter what I press.

> Do you press the crash-keys while holding down xbutton1 or does it abort right away on key press? Does this happen 100% of the time?

Yes it aborts all the time when I press any key :(

**Comment by phil294** [comment]
Ok. Could you please share your system details? E.g. Ubuntu 20.04, KDE, Linux 5.1

And I guess you're using the latest version 0.5.1?

**Comment by bpekalp** [comment]
Am I doing or did I do something wrong? Or am I missing something?

**Comment by bpekalp** [comment]
> Ok. Could you please share your system details? E.g. Ubuntu 20.04, KDE, Linux 5.1
> 
> And I guess you're using the latest version 0.5.1?

OS: Pop!_OS 22.04 LTS x86_64
Kernel: 6.2.6-76060206-generic
DE: GNOME 42.5
I use the latest version yes.

**Comment by phil294** [comment]
no, you're not doing anything wrong, but I have no idea what's going on here either. I will have to spin up a VM with your system specs next to investigate this
